src/pages/FollowUpAnalysis/Form.js

```diff
@@ -1,52 +1,18 @@
-import { useAlert } from '@dhis2/app-runtime'
 import i18n from '@dhis2/d2-i18n'
 import { Button } from '@dhis2/ui'
 import DatePicker from 'material-ui/DatePicker'
 import React from 'react'
-import AvailableDatasetsSelect from '../../components/available-datasets-select/AvailableDatasetsSelect'
-import AvailableOrganisationUnitsTree from '../../components/available-organisation-units-tree/AvailableOrganisationUnitsTree'
+import AvailableDatasetsSelect from '../../components/AvailableDatasetsSelect/AvailableDatasetsSelect'
+import AvailableOrganisationUnitsTree from '../../components/AvailableOrganisationUnitsTree/AvailableOrganisationUnitsTree'
 import cssPageStyles from '../Page.module.css'
 import jsPageStyles from '../PageStyles'
 
 /* eslint-disable react/prop-types */
```

Comment:
I was wondering why this rule is disabled.

Reply:
This rule was disabled during the app-platform port to reduce development time. I'll add the prop types for this component and the other form components.